share/spice/currency/currency.js

```diff
@@ -286,14 +278,17 @@
                 $left_select.val(from);
             }
 
-            var endpoint = "/js/spice/currency/1/" + from + "/" + to;
-            $.get(endpoint, function(payload) {
-
-                // jsonp is returned from the API so we have to alter the contents
+            $.ajax({
+                url: "/js/spice/currency/1/" + from + "/" + to,
+                beforeSend: function( xhr ) {
+                    xhr.overrideMimeType( "text/plain; charset=x-user-defined" );
```

Comment:
Both your comments are related. The api is returning a jsonp response which for one reason or another is throwing an error each time I run `$.ajax`, `$.getJSON` so I'm returning it as text and parsing into valid json. In the above line I'm overriding the MIME type of the response to plain text.